### DISCUSSION TITLE
how can i change real-time inline graph to popup in new window?

### DISCUSSION THREAD

**Discussion by rinoseo1017** [Cacti/cacti#5982] [Q&A]
Hi there,

I'm currently setting up Cacti version 1.2.27. I used the RealTime plugin (0.43) on Cacti 0.8.8 and really liked the functionality.

It's great that Cacti 1.2.27 now supports real-time graphs by default, but I'd like to view them in a separate window.
I need to monitor different types of graphs and devices in real-time simultaneously.

Are there any solutions for this?

**Comment by rinoseo1017** [comment]
i found it.

it's in 'Configuration - Users - 'user id' - User Settings - Realtime View Mode'

set Inline to New Window

**Comment by TheWitness** [comment]
Glad you found it!  You can also modify that by editing your User Profile.
